etcddb/etcddb_client.go

```diff
@@ -29,30 +29,18 @@ type EtcdClient struct {
 // PAYMENT_CHANNEL_STORAGE_CLIENT struct is not set
 func NewEtcdClient(vip *viper.Viper) (client *EtcdClient, err error) {
 
-	cluster := GetPaymentChannelCluster(vip)
-	endpoints, err := getPaymentChannelEndpoints(cluster)
-
-	if err != nil {
-		return
-	}
-
 	conf, err := GetPaymentChannelStorageClientConf(vip)
 
 	if err != nil {
 		return
 	}
 
-	if conf == nil {
-		conf = &PaymentChannelStorageClientConf{
-			ConnectionTimeout: DefaultConnectionTimeout,
-			RequestTimeout:    DefaultRequestTimeout,
-		}
-	}
+	log.WithField("etcd client conf", fmt.Sprintf("%+v", conf)).Info()
```

Comment:
Minor, but currently in code camelCase is used for log fields names, ideally I think name should be equal to the variable name to quickly find it in code

Reply:
Updated in the branch: https://github.com/stellarspot/snet-daemon/commit/1c512e6fc2dc9be24e8904a43ecc2bf48c2532b1